plus2e/AndroidProducts.mk

```diff
@@ -15,7 +15,9 @@
 #
 
 PRODUCT_MAKEFILES := \
-    $(LOCAL_DIR)/plus2e.mk
+    $(LOCAL_DIR)/plus2e.mk \
+    $(LOCAL_DIR)/plus2e_tv.mk \
```

Comment:
I think we can drop plus2e-tv product and choose type only by environment variable

Reply:
Done.